diffs/devguide74.diff

```diff
@@ -0,0 +1,820 @@
+html/en/beats/devguide/7.4/_migrating_dashboards_from_kibana_5_x_to_6_x.html
```

Comment:
diff looks good - all url diffs are the result of search/place mistake - not indicative of a real diff between files